pkg/costmodel/cluster.go

```diff
@@ -208,6 +213,9 @@ func ClusterDisks(client prometheus.Client, provider cloud.Provider, start, end
 			}
 		}
 		diskMap[key].Cost += cost
+
+		//Assigning explicitly the storage class of local storage to local
+		diskMap[key].StorageClass = "Local"
```

Comment:
Looking at the StorageClass definition [1] it looks like the `metadata.name` field can be an arbitrary string, so a user may be able to give a custom StorageClass the name `Local`. Is there a good string we could use to avoid this conflict? I know K8s maintains some limitations on strings, I just don't remember them off the top of my head.

Also, if we're going to be hardcoding strings for purposes like this, I think we should use a const, like we do for Allocation special strings: https://github.com/opencost/opencost/blob/cda202ddedb57d02bc307ea63052b29f1a504c97/pkg/kubecost/allocation.go#L21-L45

[1] https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource

Reply:
Agree, constants would be nice

Reply:
Is there a particular name you would like to differentiate local storage detected by kubecost since user can create a custom `Local` storage class? I think we can name it `__local__` per the allocation type constants but let me know. I will get all of the recommendation fixed and update this PR ASAP

Reply:
`__local__` sounds perfect to me! Thanks for digging into the resource names, that gives me confidence.